src/OrchardCore.Modules/OrchardCore.Alias/Drivers/AliasPartDisplayDriver.cs

```diff
@@ -27,6 +27,12 @@ IStringLocalizer<AliasPartDisplayDriver> localizer
             S = localizer;
         }
 
+        public override IDisplayResult Display(AliasPart part, BuildPartDisplayContext context)
+        {
+            context.Shape.Metadata.Alternates.Add($"Content__{part.ContentItem.ContentType}__{part.Alias}");
```

Comment:
This will need escaping to be a valid shape name.
Example here in the menu shapes:
https://github.com/OrchardCMS/OrchardCore/blob/ab76687ea4cc3dea68796bc224ac292f17656e60/src/OrchardCore.Modules/OrchardCore.Menu/MenuShapes.cs#L186-L226

Reply:
I've pushed an update that addresses this issue.